### DISCUSSION TITLE
In which use cases cjs bundles are usefull?

### DISCUSSION THREAD

**Discussion by rpokrovskij** [floating-ui/floating-ui#1376] [Q&A] [answered]
In which use cases cjs bundles are usefull for the popper.js? I mean why you prefer "clear CJS" to UMD?

P.S. Sorry for another "general architecture" question. I do not want to irritate you, just interesting to know the motivation of design decisions and popper architecure.

**Comment by FezVrasta** [comment, marked as answer]
I'm not completely sure to be honest, on top of my head I think Jest may use it.
